src/Tickets/Commerce/Gateways/PayPal/Assets.php

```diff
@@ -95,7 +95,7 @@ public function register() {
 				'tribe-tickets-commerce-base-gateway-checkout-js',
 				'tribe-tickets-commerce-base-gateway-checkout-toggler',
 			],
-			null,
+			'wp_enqueue_scripts',
```

Comment:
Why are we switching this to `wp_enqueue_scripts`? Seems to be unrelated to the Stripe connection box.

Reply:
I'm glad you caught that. I did that for my local environment because it wasn't loading. I did not mean to commit that change. I'll change it back.